api/available-routes.js

```diff
@@ -0,0 +1,117 @@
+const { getLogger, InputError } = require("./_utils");
+const enabledRoutesAsJson = require("../src/data/routes_1_0xc186fA914353c44b2E33eBE05f21846F1048bEda.json");
+
+const handler = async (request, response) => {
+  const logger = getLogger();
+  try {
+    let { originChainId, destinationChainId, originToken, destinationToken } =
+      request.query;
+
+    let enabledRoutes = enabledRoutesAsJson.routes.map((route) => ({
+      originChainId: String(route.fromChain),
+      originToken: String(route.fromTokenAddress),
+      destinationChainId: String(route.toChain),
+    }));
+
+    // Generate a mapping that contains similar tokens on each chain
+    // Note:  The key in this dictionary represents an l1Token address, and
+    //        the corresponding value is a nested hashmap containing a key
+    //        value pair of {chainId: l2TokenEquivalent}
+    let l1TokensToDestinationTokens = {};
```

Comment:
I don't think this is ever reassigned.
```suggestion
    const l1TokensToDestinationTokens = {};
```